model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -380,161 +244,59 @@ case class AddressResponseNag(
-  * @param paoStartNumber building number
-  * @param paoStartSuffix
-  * @param paoEndNumber
-  * @param paoEndSuffix
-  */
-case class AddressResponsePao(
-  paoText: String,
-  paoStartNumber: String,
+case class PAO(
+  text: String,
+  startNumber: String,
   paoStartSuffix: String,
   paoEndNumber: String,
   paoEndSuffix: String
 )
 
-object AddressResponsePao {
-  implicit lazy val addressResponsePaoFormat: Format[AddressResponsePao] = Json.format[AddressResponsePao]
+object PAO {
+  implicit lazy val fmt: Format[PAO] = Json.format[PAO]
 }
 
-
-/**
-  *
-  * @param saoText sub building name
-  * @param saoStartNumber sub building number
-  * @param saoStartSuffix
-  * @param saoEndNumber
-  * @param saoEndSuffix
-  */
-case class AddressResponseSao(
-  saoText: String,
-  saoStartNumber: String,
-  saoStartSuffix: String,
-  saoEndNumber: String,
-  saoEndSuffix: String
+case class SAO(
+  text: String,
+  startNumber: String,
+  startSuffix: String,
+  endNumber: String,
+  endSuffix: String
 )
 
-object AddressResponseSao {
-  implicit lazy val addressResponseSaoFormat: Format[AddressResponseSao] = Json.format[AddressResponseSao]
+object SAO {
+  implicit lazy val fmt: Format[SAO] = Json.format[SAO]
 }
 
-/**
-  * Contains address geo position
-  *
-  * @param latitude  latitude
-  * @param longitude longitude
-  * @param easting   easting
-  * @param northing  northing
-  */
-case class AddressResponseGeo(
+case class GEO(
   latitude: Double,
   longitude: Double,
-  easting: Int,
-  northing: Int
+  easting: Double,
+  northing: Double
 )
 
-object AddressResponseGeo {
-  implicit lazy val addressResponseGeoFormat: Format[AddressResponseGeo] = Json.format[AddressResponseGeo]
+object GEO {
+  implicit lazy val fmt: Format[GEO] = Json.format[GEO]
 }
 
-/**
-  * Contains response status
-  *
-  * @param code    http code
-  * @param message response description
-  */
-case class AddressResponseStatus (
+case class Status(
   code: Int,
   message: String
 )
 
-object AddressResponseStatus {
-  implicit lazy val addressResponseStatusFormat: Format[AddressResponseStatus] = Json.format[AddressResponseStatus]
+object Status {
+  implicit lazy val fmt: Format[Status] = Json.format[Status]
 }
 
-object OkAddressResponseStatus extends AddressResponseStatus(
-  code = Status.OK,
-  message = "Ok"
-)
-
-object NotFoundAddressResponseStatus extends AddressResponseStatus(
-  code = Status.NOT_FOUND,
-  message = "Not Found"
-)
-
-object BadRequestAddressResponseStatus extends AddressResponseStatus(
-  code = Status.BAD_REQUEST,
-  message = "Bad request"
-)
-
-
-/**
-  * Contains one response error
-  *
-  * @param code    error code
-  * @param message error description
-  */
-case class AddressResponseError(
+case class Error(
   code: Int,
   message: String
 )
 
-object AddressResponseError {
-  implicit lazy val addressResponseErrorFormat: Format[AddressResponseError] = Json.format[AddressResponseError]
+object Error {
```

Comment:
That would imply that you'd need to provide this context `AddressResponse.Error` every time that you are using this class. Thus this would apply artificial restrictions that nobody is forced to follow and will simply forget. By adding the prefix `AddressResponse` to the class, the context becomes mandatory.